central/clusterinit/service/service_impl.go

```diff
@@ -129,6 +130,10 @@ func (s *serviceImpl) GenerateInitBundle(ctx context.Context, request *v1.InitBu
 }
 
 func (s *serviceImpl) GenerateCRS(ctx context.Context, request *v1.CRSGenRequest) (*v1.CRSGenResponse, error) {
+	if !features.ClusterRegistrationSecrets.Enabled() {
+		return nil, status.Error(codes.Unimplemented, "Support for generating Cluster Registration Secrets (CRS) is not enabled")
```

Comment:
```suggestion
		return nil, status.Error(codes.Unimplemented, "support for generating Cluster Registration Secrets (CRS) is not enabled")
```

Reply:
Either it's either a lower case start according to the convention for errors, or a sentence and then it should end with a full stop?